packages/plugins/audit/package.json

```diff
@@ -34,13 +34,15 @@
     "@verdaccio/core": "workspace:8.0.0-next-8.18",
     "express": "4.21.2",
     "https-proxy-agent": "5.0.1",
-    "node-fetch": "cjs"
+    "node-fetch": "3.3.2"
```

Comment:
`cjs: 2.6.7` change this will break 6.x, since EMS is mandatory, I'd not touch this at this moment.